teamB/JeongWooJin/controllers/likeController.js

```diff
@@ -0,0 +1,16 @@
+const likeService = require('../services/likeService');
+
+const pushLikes = async (req, res) => {
```

Comment:
함수 이름을 액션과 대상으로 잘 지어주셨습니다.
조그만 수정을 하자면, 좋아요를 누르는 상화이긴 하지만 대상이 되는 데이터의 입장에서 보면 좋아요 데이터가 생성되는 것이기 때문에 다음과 같이 수정해 주면 좋을 것 같습니다.

```suggestion
const createLike = async (req, res) => {
```